packages/prisma/index.ts

```diff
@@ -2,6 +2,9 @@ import { Prisma, PrismaClient } from "@prisma/client";
 
 import { bookingReferenceMiddleware, eventTypeDescriptionParseAndSanitizeMiddleware } from "./middleware";
 
+// REVIEW: This enum is needed in apps/website to generate the sitemap paths
+export { AppCategories } from "@prisma/client";
+
```

Comment:
@PeerRich I _think_ this is needed in order to make the sitemap work https://github.com/calcom/website/pull/420

Reply:
Why don't we import directly from prisma/client in website?

Reply:
That was my first try. But while TS was happy, I ran into the following error:
`import { AppCategories } from "@calcom/prisma/client";`
<img width="1393" alt="CleanShot 2023-01-27 at 20 51 57@2x" src="https://user-images.githubusercontent.com/18185649/215226990-68f7368a-4286-4068-a27b-9248f2a1c0cd.png">

When I import it from the barrel file (`import { prisma, AppCategories } from "@calcom/prisma";`) it's `undefined`:
<img width="343" alt="CleanShot 2023-01-27 at 21 03 47@2x" src="https://user-images.githubusercontent.com/18185649/215227983-480c6586-7e6a-4163-8b38-e67aeea4c6a6.png">

Reply:
`import { AppCategories } from "@prisma/client";``

Reply:
I just tried the [transpilePackages](https://beta.nextjs.org/docs/api-reference/next.config.js#transpilepackages) (supposed to replace next-transpile-modules) but it has the same result as above.

```ts
  transpilePackages: [
    "@calcom/app-store",
    "@calcom/dayjs",
    "@calcom/emails",
    "@calcom/trpc",
    "@calcom/embed-core",
    "@calcom/embed-react",
    "@calcom/features",
    "@calcom/lib",
    "@calcom/prisma",
    "@calcom/ui",
  ],
```

Reply:
I have worked with a prisma generator in a different app that generated a `prisma-types` package for all prisma types, that we're reused in apps.

Also possible, but pollutes prisma's codegen pipeline a little.

Other than these two approaches, I'm not entirely sure how to make this work.

@zomars do you have another idea maybe? Or any preference regarding the prisma generator vs. polluting `@calcom/prisma`'s barrel file?